help: New mobile instructions for Flutter launch.

laurynmm opened the pull request:
While making updates for #34748, I noticed a couple of places we can add documentation for the Flutter mobile app's launch:
- **Zulip Version**: Seemed useful for potentially handling support issues to be able to see the version of the app that the user is running.
- **Follow a topic**: The Flutter app has implemented following and unfollowing topics in the topic action menu.

---

**Screenshots and screen captures:**

[CURRENT DOCUMENTATION](https://zulip.com/help/view-zulip-version)
![Screenshot from 2025-06-16 13-07-24](https://github.com/user-attachments/assets/8df44572-8f0f-43da-8bf2-6ecfcc4a74ba)

[CURRENT DOCUMENTATION](https://zulip.com/help/follow-a-topic#follow-or-unfollow-a-topic)
![Screenshot from 2025-06-13 21-01-14](https://github.com/user-attachments/assets/0d5e3edc-b9e2-475d-9c5e-87f57e4c8fb7)

[CURRENT DOCUMENTATION](https://zulip.com/help/follow-a-topic#manage-configured-topics)
![Screenshot from 2025-06-16 13-03-49](https://github.com/user-attachments/assets/2f4c7de1-7597-4b02-8207-1b7f9a5ef8a3)

---

<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [ ] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [ ] Each commit is a coherent idea.
- [ ] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [ ] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.
</details>

chrisbobbe commented on help/follow-a-topic.md:
```diff
@@ -112,6 +118,19 @@ messages just in followed topics, without counting other channel messages.
+
+1. Select a channel containing topics you want to configure.
+
+1. Tap **Topics** in the upper right corner of the app.
```
We plan to replace the "TOPICS" text button with an icon: https://github.com/zulip/zulip-flutter/issues/1532 I'm OK with this commit being merged as-is, and I'll make a note on the zulip-flutter issue to update this doc.

Also nit: the current button "TOPICS" is in all caps.

alya replied:
Our pattern for all-caps font has been to use all-lowercase in the help center. E.g., "views" in https://zulip.com/help/left-sidebar.

chrisbobbe reviewed (commented):
This is great, thanks for finding these!! One small comment below.

alya commented:
I don't think it makes sense to have a "View Zulip mobile app version" heading in the desktop app subsection. We should make an new top-level "Zulip mobile app version" section; we can basically have 2 copies of the same intro sentence for the mobile and desktop sections.

laurynmm commented:
**Updates**:
- In the "Manage configured topics" section, updated **Topics** to **topics** as that's our convention for all-caps UI in the help center.
- Updated the changes to "View Zulip version" to have a separate section for the mobile apps. And added the new mobile installation guide to the related articles section.

Screenshots are updated in the main PR comment above.

timabbott commented:
Merged, thanks @laurynmm!
